1209/D10D/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Piunora
+owner: Diodes Delight
```

Comment:
```suggestion
owner: Diodes-Delight
```

Reply:
If due formatting restrictions thats fine! otherwise its not written with a -

Reply:
It has to match the filename in `org/`

Reply:
(That's why the CI is failing.)